br/pkg/lightning/backend/external/iter.go

```diff
@@ -229,6 +229,8 @@ func (i *mergeIter[T, R]) next() bool {
 				// currently only one reader will become hotspot
 				if cnt > (checkHotspotPeriod / 2) {
 					i.lastHotspotIdx = idx
+
+					logutil.BgLogger().Info("new hotspot reader", zap.Any("idx", i.lastHotspotIdx), zap.Any("cnt", cnt))
```

Comment:
I remember the comment of  `Any` says it's a bit slower, we can use zap.Int64 instead

Reply:
It's for debug, I forgot to remove it